package io.github.blyznytsiaorg.bibernate.exception;

public class BibernateDataSourceException extends RuntimeException {
    public BibernateDataSourceException(String message, Throwable cause) {
        super(message, cause);
    }
}
